src/NuGet.Core/NuGet.Commands/CommandRunners/PackCommandRunner.cs

```diff
@@ -733,7 +733,7 @@ private PackageArchiveReader BuildFromProjectFile(string path)
                 }
 
                 if (packageArchiveReader != null && !_packArgs.NoPackageAnalysis)
-                {
+                {   
```

Comment:
I thought our .`editorconfig` settings should prevent trailing whitespace :(

In any case, remove it.